### PR TITLE
[RFR] New test test_control_crud_ansible_playbook_action

### PR DESCRIPTION
**Pull request by quarckster**
Intent
=================

{{pytest: -v -k "test_control_crud_ansible_playbook_action" --long-running}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 9617d0c683d9fa171fcf948a2a27fc8821b8421d:

`cfme/tests/ansible/test_embedded_ansible_basic.py`:
- :heavy_exclamation_mark: Line 206:1: **W391** *blank line at end of file*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: cfme/tests/ansible/test_embedded_ansible_basic.py [hunk 1]
```diff
@@ -139,3 +170,59 @@ def test_embed_tower_playbooks_list_changed(wait_for_ansible):
         playbooks.append(repository.playbooks.all())
         repository.delete()
     assert not set(playbooks[1]).issuperset(set(playbooks[0]))
+
+
+@pytest.mark.tier(2)
+def test_control_crud_ansible_playbook_action(request, catalog_item):
```
Comment (mkoura) [inline cfme/tests/ansible/test_embedded_ansible_basic.py]: Please add docstring.

File: cfme/tests/ansible/test_embedded_ansible_basic.py [hunk 2]
```diff
@@ -139,3 +170,59 @@ def test_embed_tower_playbooks_list_changed(wait_for_ansible):
...
+            }
+        }
+    view = navigate_to(action, "Edit")
+    assert view.description.value == new_descr
+    assert view.run_ansible_playbook.inventory.hosts.value == ipaddr
+    action.delete()
+
+
+@pytest.mark.tier(2)
+def test_control_add_ansible_playbook_action_invalid_address(request, catalog_item):
```
Comment (mkoura) [inline cfme/tests/ansible/test_embedded_ansible_basic.py]: Please add docstring.

**Pull request closed without merging**